js2/10.js

```diff
@@ -0,0 +1,18 @@
+//After each step, log the value or question / answer.
+
+//Create a const named cats that is an array with the following strings: Siamese, Calico, and Maine Coon
+const cats = ['Siamese', 'Calico', 'Maine Coon'];
+console.log(cats);
+//Using indexes, display the first element of the cats array.
+console.log(cats[0]);
+//Create a const named aggregate that contains the same values as #9
+const alpha = [];
+const beta = [3, 4, 10];
+const delta = ['hey', 'there']
+
+const aggregate = [alpha, beta, delta];
```

Comment:
```
[ [], [ 3, 4, 10 ], [ 'hey', 'there' ] ]
```
👍 

I think I may have intended something slightly different, but the way the original issue was phrased this is correct.